packages/components/badge/src/BadgeItem.tsx

```diff
@@ -0,0 +1,79 @@
+import { forwardRef, HTMLAttributes, PropsWithRef } from 'react'
+
+import { styles, type StylesProps } from './BadgeItem.styles'
+import {
+  DEFAULT_INTENT,
+  DEFAULT_LABELS,
+  DEFAULT_OVERFLOW_COUNT,
+  DEFAULT_SIZE,
+  DEFAULT_TYPE,
+} from './config'
+
+interface ACCESSIBILITY_LABELS {
```

Comment:
IMO interfaces should be `CamelCase` and written in singular

Reply:
Yep, I treated it as a constant, but I think we all should agree on it and write the conclusions somewhere...